lib/src/optional/default_optional_update_service.dart

```diff
@@ -26,13 +26,14 @@ class DefaultOptionalUpdateService implements OptionalUpdateService {
             iau.UpdateAvailability.updateAvailable;
       } else if (Platform.isIOS && updateConfig.appStoreAppId != null) {
         // Query AppStore
-        final iTunesResults = await ITunesSearchAPI().lookupById(
+        final iTunesSearchAPI = ITunesSearchAPI();
+        final iTunesResults = await iTunesSearchAPI.lookupById(
           updateConfig.appStoreAppId!,
           country: updateConfig.appstoreCountryCode,
         );
         if (iTunesResults != null) {
           // Parse AppStore response
-          final iTunesVersionString = ITunesResults.version(iTunesResults);
+          final iTunesVersionString = iTunesSearchAPI.version(iTunesResults);
```

Comment:
Shouldn't this be `iTunesResults` instead?

Reply:
@JTorrus `iTunesSearchAPI.version()` this method was static in old version of `upgrader` package but now ITunesResults is `extension ITunesResults on ITunesSearchAPI ` and `version()` is a member method of `ITunesSearchAPI`